pkg/handler/handler.go

```diff
@@ -335,6 +337,37 @@ func (h handler) apbAddSpec(w http.ResponseWriter, r *http.Request, params map[s
 	writeDefaultResponse(w, http.StatusOK, resp, err)
 }
 
+func (h handler) apbRemoveSpec(w http.ResponseWriter, r *http.Request, params map[string]string) {
+	ansibleBroker, ok := h.broker.(broker.DevBroker)
+	if !ok {
+		h.log.Errorf("unable to use broker - %T as ansible service broker", h.broker)
```

Comment:
It's actually not clear to me why there is a *f variant.

https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/svc_acct.go#L127

Is an example of the non *f variant still accepting a format string. The docs literally use the same docs string: https://godoc.org/github.com/op/go-logging#Logger.Error

Anyone else have an opinion here?